cilium: tunnel was deprecated in v1.14 and has been removed in v1.15.

action url: https://github.com/spidernet-io/spiderpool/actions/runs/11059440877

```
install-default-cni.sh : E2E_CLUSTER_NAME spiderpool0926201300 
install-default-cni.sh : INSTALL_CALICO false 
install-default-cni.sh : INSTALL_CILIUM true 
install-default-cni.sh : CLUSTER_PATH /home/runner/work/spiderpool/spiderpool/test/.cluster/spiderpool0926201300 
 ===> Start to install cilium 
CILIUM_HELM_OPTIONS:  --set cni.exclusive=false                             --set kubeProxyReplacement=false                             --set k8sServiceHost=spiderpool0926201300-control-plane                             --set k8sServicePort=6443                             --set bpf.vlanBypass={0}  --set ipam.operator.clusterPoolIPv6PodCIDRList=fd00:10:244::/112                                    --set ipv4.enabled=false                                    --set ipv6.enabled=true                                    --set tunnel=disabled                                    --set ipv6NativeRoutingCIDR=fd00:10:244::/112                                    --set autoDirectNodeRoutes=true                                    --set enableIPv6Masquerade=true                                    --set routingMode=native        --set image.repository=quay.io/cilium/cilium       --set image.useDigest=false       --set certgen.image.repository=quay.io/cilium/certgen       --set hubble.relay.image.repository=quay.io/cilium/hubble-relay       --set hubble.relay.image.useDigest=false       --set hubble.ui.backend.image.repository=quay.io/cilium/hubble-ui-backend       --set hubble.ui.frontend.image.repository=quay.io/cilium/hubble-ui       --set etcd.image.repository=quay.io/cilium/cilium-etcd-operator       --set operator.image.repository=quay.io/cilium/operator        --set operator.image.useDigest=false        --set preflight.image.repository=quay.io/cilium/cilium       --set preflight.image.useDigest=false       --set nodeinit.image.repository=quay.io/cilium/startup-script 
"cilium" has been added to your repositories
Hang tight while we grab the latest from your chart repositories...
...Successfully got an update from the "cilium" chart repository
Update Complete. ⎈Happy Helming!⎈
Error: execution error at (cilium/templates/validate.yaml:33:5): tunnel was deprecated in v1.14 and has been removed in v1.15. For details please refer to https://docs.cilium.io/en/v1.15/operations/upgrade/#helm-options

Use --debug flag to render out invalid YAML
make[2]: *** [Makefile:37: kind-init] Error 1
make[2]: Leaving directory '/home/runner/work/spiderpool/spiderpool/test'
make[1]: *** [Makefile:299: e2e_init] Error 2
make[1]: Leaving directory '/home/runner/work/spiderpool/spiderpool'
make: *** [Makefile:336: e2e_init_cilium_legacyservice] Error 2
```

--- a/test/scripts/install-default-cni.sh
+++ b/test/scripts/install-default-cni.sh
@@ -191,7 +191,6 @@ function install_cilium() {
             CILIUM_HELM_OPTIONS+=" --set ipam.operator.clusterPoolIPv6PodCIDRList=${CILIUM_CLUSTER_POD_SUBNET_V6} \
                                    --set ipv4.enabled=false \
                                    --set ipv6.enabled=true \
-                                   --set tunnel=disabled \
                                    --set ipv6NativeRoutingCIDR=${CILIUM_CLUSTER_POD_SUBNET_V6} \
                                    --set autoDirectNodeRoutes=true \
                                    --set enableIPv6Masquerade=true \

